ts/utils/SessionManager.ts

```diff
@@ -85,6 +85,28 @@ export class SessionManager<T> {
    */
   public get = () => fromNullable(this.token);
 
+  /**
+   * Returns a new token
+   */
+  public getNewToken = () => async () => {
+    let count = 0;
+    while (count <= this.maxRetries) {
+      count += 1;
+      this.exclusiveTokenUpdate(true);
```

Comment:
```suggestion
      await this.exclusiveTokenUpdate(true);
```

Reply:
Fixed in [2196845](https://github.com/pagopa/io-app/pull/2490/commits/2196845b7a6609c46ba0c52b2e199121edbedadb)